cleanlab_studio/internal/api/api.py

```diff
@@ -699,7 +701,11 @@ async def tlm_get_confidence_score(
             res = await client_session.post(
                 f"{tlm_base_url}/get_confidence_score",
                 json=dict(
-                    prompt=prompt, response=response, quality=quality_preset, options=options or {}
+                    prompt=prompt,
+                    response=response,
+                    input_metadata=input_metadata or {},
```

Comment:
is input_metadata being validated anywhere?

Reply:
It is constructed/validated [here](https://github.com/cleanlab/cleanlab-studio/blob/94bcf094a12db7897eb87477f80e0cf03b875a4a/cleanlab_studio/studio/trustworthy_language_model.py#L519) (among other places where there are `kwargs` inputs)

Reply:
I see it is treated as a kwarg. That part LGTM then!